### ISSUE TITLE
deployApp() failure due to check_exclusive?

### ISSUE THREAD

**Issue by rdewald** [rstudio/rsconnect#693]
We are unable to publish today after recent commits.  Our developers see some version of this error:

Error in `rsconnect::deployApp()`:
! Exactly one of `appFiles` or `appFileManifest` must be supplied.

@ryanduggan submitted this observation as part of the debug process.

> In the call to rsconnect:::standardizeAppFiles on line 247. the first thing standardizeAppFiles does is call rlang::check_exclusive to make sure only one of appFiles and appFileManifest is supplied. I just tested that function, and supplying one of those arguments as NULL still counts as supplying it, so check_exclusive will always fail.

Thanks for the attention.

**Comment by aronatkins** [comment]
This may be addressed by #684, which removes the call to `check_exclusive`.

**Comment by ryanduggan** [comment]
Thanks @aronatkins , it looks removing the call to `check_exclusive` does rectify the issue we were seeing.

**Comment by hadley** [comment]
@rdewald BTW why aren't you using the CRAN version of rsconnect?

**Comment by rdewald** [comment]
@hadley because of a naive architecture design fail.

We use a "shared baseline" collaboration strategy via a frozen RSPM repo for CRAN.  I found that some small number of posit-maintained packages needed updating from our frozen CRAN versions so we could advance application releases for workbench and connect.

Being too cute, I identified those and made a github-scraping repo, naively assuming commits to main wouldn't break a current release of Posit software.  I was wrong about that.

I have opted instead to have a cran-curated list of those packages (and their dependencies) available as a repo in place of my too-clever Github based architecture.  That's why I closed this.
